### PR TITLE
DTS release v1.2.8

### PR DESCRIPTION
**Pull request by TomaszAIR**

### CODE REVIEW

File: docs/dasharo-tools-suite/documentation.md [hunk 1]
```diff
@@ -268,8 +271,11 @@ This section describes the functionality of the Dasharo Tools Suite. These are:
 DTS can be used to flash Dasharo firmware on your hardware. To achieve this,
 boot DTS, choose option number `2`. After creating
 [report](../glossary.md#dasharo-hardware-compatibility-list-report) with
-firmware dump as backup, type `p` to confirm the installation of Dasharo
-firmware.
+firmware dump as backup, type `d` or `c` to confirm the installation of Dasharo
+firmware. Option `c` stands for community release which is available for anyone
+using Dasharo Tools Suite, option `d` stands for
+[DES](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-entry-subscription-subscriber)
+release and it is only available for Dasharo Entry Subscription holders.
```
Comment (miczyg1) [inline docs/dasharo-tools-suite/documentation.md:278]:
```suggestion
release and it is only available for Dasharo Entry Subscription subscribers.
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/dasharo-tools-suite/documentation.md [hunk 2]
```diff
@@ -372,7 +380,10 @@ Please consider the following options depending on your situation:
 ### Firmware update
 
 DTS can be used to update Dasharo firmware. To achieve this, boot it on platform
-with flashed Dasharo and choose option number `5`.
+with flashed Dasharo and choose option number `5`. You may see additional
+information about available updates if you are not [Dasharo Entry
+Subscription](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-entry-subscription-subscriber)
+holder.
```
[the remark this answers is not in the document]
Comment (TomaszAIR) [reply docs/dasharo-tools-suite/documentation.md:386]: @miczyg1 good to know, thanks.

File: docs/dasharo-tools-suite/documentation.md (hunk 1 above)
Comment (TomaszAIR) [reply docs/dasharo-tools-suite/documentation.md]: @miczyg1 fixed https://github.com/Dasharo/docs/pull/648/commits/48f39121ae7b4260b660d7a090d404aa713aa884#diff-8cfee4c026a23b6eea51be5305d2c9db7d12139283fcec2ee3522ffffc0dc026R278

File: docs/dasharo-tools-suite/documentation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (TomaszAIR) [reply docs/dasharo-tools-suite/documentation.md]: @miczyg1 fixed https://github.com/Dasharo/docs/pull/648/commits/48f39121ae7b4260b660d7a090d404aa713aa884#diff-8cfee4c026a23b6eea51be5305d2c9db7d12139283fcec2ee3522ffffc0dc026R386

File: docs/dasharo-tools-suite/documentation.md [hunk 3]
```diff
@@ -288,6 +294,8 @@ This feature is supported on the following platforms:
 * Dell OptiPlex 7010/9010,
 * MSI PRO Z690-A DDR4,
 * MSI PRO Z690-A DDR5,
+* MSI PRO Z790-A DDR4,
+* MSI PRO Z790-A DDR5,
```
Comment (miczyg1) [inline docs/dasharo-tools-suite/documentation.md:298]:
```suggestion
* MSI PRO Z790-P DDR4,
* MSI PRO Z790-P DDR5,
```

File: docs/dasharo-tools-suite/documentation.md [hunk 4]
```diff
@@ -268,8 +271,11 @@ This section describes the functionality of the Dasharo Tools Suite. These are:
 DTS can be used to flash Dasharo firmware on your hardware. To achieve this,
 boot DTS, choose option number `2`. After creating
 [report](../glossary.md#dasharo-hardware-compatibility-list-report) with
-firmware dump as backup, type `p` to confirm the installation of Dasharo
-firmware.
+firmware dump as backup, type `d` or `c` to confirm the installation of Dasharo
+firmware. Option `c` stands for community release which is available for anyone
+using Dasharo Tools Suite, option `d` stands for
+[DES](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-entry-subscription-subscriber)
+release and it is only available for Dasharo Entry Subscription subscribers.
```
Comment (miczyg1) [inline docs/dasharo-tools-suite/documentation.md:278]:
```suggestion
release and it is only available to Dasharo Entry Subscription subscribers.
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/dasharo-tools-suite/documentation.md [hunk 5]
```diff
@@ -11,6 +11,9 @@ that it boots on the following platforms:
   report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=0&range=A75)),
 * MSI PRO Z690-A DDR5 ([test
   report](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit#gid=777478871&range=A81)),
+* MSI PRO Z790-A DDR4,
+* MSI PRO Z790-A DDR5 ([test
```
Comment (miczyg1) [inline docs/dasharo-tools-suite/documentation.md:15]:
```suggestion
* MSI PRO Z790-P DDR4,
* MSI PRO Z790-P DDR5 ([test
```

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**